examples/log_files.yml.example.advanced

```diff
@@ -5,7 +5,8 @@ files:
   - /opt/misc/*.log
   - /home/**/*.log
   - /var/log/mysqld.log
-  - /var/run/mysqld/mysqld-slow.log
+  - path: /var/run/mysqld/mysqld-slow.log
+    tag: slow_queries.log
```

Comment:
Just tried this locally to see what it looks like. I sort of feel like it might be better to have something like `mysql_slow_queries` (or something shorter if anything comes to mind). Reasoning:

* Having a `.log` extension may lead to someone forgetting that this isn't a real file
* Including the app name makes it easier to know where it comes from, although other files may break that rule so feel free to ignore this point if you think the new name is a tad long